docs/ruleset_functions/get_dict_of_zones_and_terminal_units_served_by_hvac_sys(hvac.id).md.md

```diff
@@ -0,0 +1,40 @@
+# get_dict_of_zones_and_terminal_units_served_by_hvac_sys    
+
+**Description:** Returns a dictionary of zones and terminal unit IDs associated with each HVAC system in the RMR.   
+
```

Comment:
@claperle I am unclear on how this function would be used. I dont see this in the function for baseline system 5, #540 nor any of the single zone systems.

I will be able to provide a more constructive review after we discuss how this function will be used.

Reply:
@supriyagoel CML: This function is run in the master function, [get_baseline_system_types()](https://gcc02.safelinks.protection.outlook.com/?url=https%3A%2F%2Fgithub.com%2Fpnnl%2Fruleset-checking-tool%2Fpull%2F223&data=05%7C01%7CSupriya.Goel%40pnnl.gov%7C8128759c3bc048aa848f08da6990543e%7Cd6faa5f90ae240338c0130048a38deeb%7C0%7C0%7C637938365913789117%7CUnknown%7CTWFpbGZsb3d8eyJWIjoiMC4wLjAwMDAiLCJQIjoiV2luMzIiLCJBTiI6Ik1haWwiLCJXVCI6Mn0%3D%7C3000%7C%7C%7C&sdata=ZuzK9vkgWJneOkUN%2BvhlgBb%2FTOyuIPixoyFlDX6cOYw%3D&reserved=0), which loops through each HVAC system in the B_RMR.  The master function starts by calling this function to create a dictionary with a list of terminal units and zones associated with each hvac system in the RMR. As the master loops through each hvac system it creates a list of terminal units and a list of zones associated with the hvac system from the dictionary that was created. The list of zones and list of terminal units is then sent to various functions as needed. This is designed to maximum code efficiency.